backend/de.metas.business/src/test/java/de/metas/business/BusinessTestHelper.java

```diff
@@ -393,4 +395,13 @@ public static OrgId createOrgWithTimeZone()
 	{
 		return AdempiereTestHelper.createOrgWithTimeZone();
 	}
+
+	public I_PP_Product_BOMVersions createBOMVersions(final ProductId productId)
+	{
+		final I_PP_Product_BOMVersions bomVersions = InterfaceWrapperHelper.newInstance(I_PP_Product_BOMVersions.class);
```

Comment:
same here. we need to know the org, to make sure not to end up with some unexpected org, just because that'S the current user's org

Reply:
it's just a test class, it's done the right way in `org.eevolution.api.impl.ProductBOMVersionsDAO#createBOMVersions` && `org.eevolution.api.impl.ProductBOMDAO#createBOM`

Reply:
right, sry. sometimes hard to see the picture when looking at lines